frontend/src/main.ts

```diff
@@ -58,7 +58,7 @@ const apiUrl = `${protocol}://${import.meta.env.VITE_API_URL}${port}`;
 app.provide('apiUrl', apiUrl);
 app.provide('bookingUrl', `${protocol}://${import.meta.env.VITE_BASE_URL}/appointments/all/`);
 
-const loc = localStorage?.getItem('locale') ?? (navigator.language || navigator.userLanguage);
```

Comment:
I couldn't find any documentation on a `navigator.userLanguage` property, so I removed it 🤷🏻

Reply:
Apparently it's IE only 😅

Reply:
Haha I see, then I guess it's safe to remove it 😅